scene/gui/graph_edit.cpp

```diff
@@ -1878,15 +1935,19 @@ void GraphEdit::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "zoom_min"), "set_zoom_min", "get_zoom_min");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "zoom_max"), "set_zoom_max", "get_zoom_max");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "zoom_step"), "set_zoom_step", "get_zoom_step");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "show_zoom_label"), "set_show_zoom_label", "is_showing_zoom_label");
 
 	ADD_GROUP("Minimap", "minimap_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "minimap_enabled"), "set_minimap_enabled", "is_minimap_enabled");
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "minimap_size", PROPERTY_HINT_NONE, "suffix:px"), "set_minimap_size", "get_minimap_size");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "minimap_opacity"), "set_minimap_opacity", "get_minimap_opacity");
 
 	ADD_GROUP("UI", "");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "arrange_nodes_button_hidden"), "set_arrange_nodes_button_hidden", "is_arrange_nodes_button_hidden");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "hide_menu"), "set_hide_menu", "is_hiding_menu");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "show_zoom_label"), "set_show_zoom_label", "is_showing_zoom_label");
```

Comment:
IMO these properties should either be all negative or all positive. With the compatibility methods this shouldn't be a problem (introducing methods for show_zoom_label [all negative] or change the ones you already wrote [all positive]).
Beyond that I have no opinion on which (all positive or all negative) is better.

Reply:
I understand the appeal, and I would support it personally. But there was an alternative POV that checkboxes should be off by default, and any alternative behavior that you opt into should be enabled by toggling them. So because preserving the name would be slightly more compatibility friendly, I decided to go in the direction of that alternative POV.

I also considered that switching the polarity is a more annoying compat change than a simple rename. You can't just search and replace it, you have to invert the logic everywhere. So for better or worse, I'd stick to the white raven `show_zoom_label` being what it is.

But ultimately, I'm fine either way, as long as we change `arrange_nodes_button_hidden` 🙃

Reply:
I was arguing for off-by-default checkboxes, but to be honest I'm not that convinced that it's the absolute best design. I think here a `show_menu` that's enabled by default would also be ok.

So pick what you prefer, and it should be fine :)

Reply:
Alright, the I guess it would make more sense to invert `arrange_nodes_button_hidden` and keep every property as `show_*` with some being on and some being off by default.